test-automation-school/src/test/java/test/automation/school/HWTest.java

```diff
@@ -0,0 +1,87 @@
+package test.automation.school;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import test.automation.school.pageobjects.*;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Created by Max on 5/17/2017.
+ */
+public class HWTest extends BaseTest {
+    HomePage homePage;
+    SearchPage searchPage;
+    SearchResultsPage searchResultsPage;
+
+    @Test
+    public void ssLvSearchTest() {
+        // 1. Открыть браузер и развернуть на весь экран.
+        // 2. Зайти на ss.lv.
+        homePage = new HomePage(driver);
+
+        // 3. Поменять язык на русский.
+        homePage.switchLangTo("RU");
+
+        //3a. navigate to Search page
+        homePage.pageHeaderSearch.click();
```

Comment:
As mentioned, try not to expose WebElements from Page Objects, use methods instead to interact with the page.